shoppingfeed-cdiscount-fees/src/Admin/Order.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace ShoppingFeed\ShoppingFeedWCCdiscountFees\Admin;
+
+use ShoppingFeed\ShoppingFeedWC\Addons\Marketplace;
+use ShoppingFeed\ShoppingFeedWC\Addons\Shipping\Marketplaces\Cdiscount;
+use ShoppingFeed\ShoppingFeedWC\Orders\Order as SFDefaultOrder;
+
+class Order {
+
+    use Marketplace;
+
+    /**
+     * @var $sf_cdiscount_fee_meta_key
+     */
+    private $sf_discount_fee_meta_key = 'sf_cdiscount_fees';
+
+    /**
+     * @var $sf_cdiscount_fee_meta_value
+     */
+    public $sf_cdiscount_fee_meta_value;
+
+    public function __construct() {
+		add_filter( 'sf_pre_add_fees', [ $this, 'save_cdiscount_fees_as_meta' ], 10, 2 );
+    }
+
+    /**
+     * Get the order fees meta value
+     *
+     * @return mixed
+     */
+    public function get_cdiscount_fee_meta_value() {
+        return $this->sf_cdiscount_fee_meta_value;
+    }
+
+	/**
+	 * Save the Cdiscount fees as WC order meta data
+	 *
+	 * @param $wc_order
+	 * @param $sf_order
+	 * @param $fees
+	 *
+	 * @return bool
+	 */
+	private function save_cdiscount_fees_as_meta( $wc_order, $sf_order, $fees ) {
```

Comment:
@jeremykervran les paramètres ne correspondent pas à ce que le filtre core envoi `apply_filters( 'sf_pre_add_fees', false, $wc_order, $this, $this->fees );`